executor/runtime/docker/docker.go

```diff
@@ -987,7 +996,7 @@ func (r *DockerRuntime) Prepare(ctx context.Context) error { // nolint: gocyclo
 				"TITUS_SECCOMP_AGENT_HANDLE_PERF_SYSCALLS": "true",
 			})
 		}
-		if r.c.SeccompAgentEnabledForNetSyscalls() {
+		if r.c.SeccompAgentEnabledForNetSyscalls() || r.c.EffectiveNetworkMode() == titus.NetworkConfiguration_Ipv6AndIpv4Fallback.String() {
```

Comment:
Long term, should SeccompAgentEnabledForNetSyscalls go away?

Reply:
Yes, I just didn't want to break existing stuff.

Oh I see, no you are right, EffectiveNetworkMode() knows about that already.